zml/module.zig

```diff
@@ -95,12 +96,14 @@ pub const CompilationContext = struct {
     }
 
     pub fn activate(self: *CompilationContext) void {
+        self._previous = _current;
```

Comment:
wouldn't it be safer to always spinup a new thread for compilation ?

Reply:
It would, but that interface would be at a higher level, not in `CompilationContext`